src/heuristics/Doc2Vec.py

```diff
@@ -0,0 +1,68 @@
+import string
+
+import nltk
+import numpy as np
+from gensim.models.doc2vec import Doc2Vec
+from nltk.corpus import stopwords
+
+from heuristics.Heuristics import AbstractHeuristic
+
+
+class Doc2VecHeuristic(AbstractHeuristic):
+    def __init__(self, cleaned):
+        self.model = Doc2Vec.load("../models/" + ("doc2vec_cleaned" if cleaned else "doc2vec"))
+        self.api = None
+        self.goal_vector = None
+        self.summaries = {}
+        self.goal = None
+        self.cleaned = cleaned
+
+    def setup(self, api, start, goal):
+        self.api = api
+        goal_summary_array = self.get_summary_array(goal)
+        self.goal_vector = self.model.infer_vector(doc_words=goal_summary_array, alpha=0.025, steps=20)
```

Comment:
Are these values hand-tuned? Do they need to match how the model was trained? It feels a bit weird to have the numbers hard coded here.

Reply:
I read on stack overflow that these were better parameters than the default. Not 100% sure what they do.